src/main/java/eu/usrv/enhancedlootbags/core/items/ItemLootBag.java

```diff
@@ -66,16 +69,15 @@ public int getItemEnchantability() {
     @SideOnly(Side.CLIENT)
     public void registerIcons(IIconRegister pIconRegister) {
         _mIcoDefault = pIconRegister.registerIcon(String.format("%s:lootbag_generic", EnhancedLootBags.MODID));
-        // for (LootGroup tGrp : _mLGHandler.getLootGroups().getLootTable())
-        // tGrp.setGroupIcon(pIconRegister.registerIcon(String.format("%s:lootbags/lootbag_%d",
-        // Refstrings.MODID, tGrp.mGroupID)));
+        for (LootGroup tGrp : _mLGHandler.getLootGroups().getLootTable()) {
```

Comment:
can loot groups change at runtime? if yes I'd say we should clear the _mGroupIcons map before updating it

Reply:
There certainly is a CLI to update the groups and I see that the texture manager can be reloaded, restarting the entire icon registration process. Nice catch.

I'll note that I haven't added CLI support for images because it wasn't a use case that made sense to me, given that we would need to handle invalid texture definitions in a non-crashy way and that you need to add your textures to a jar anyway.

Reply:
we do have txloader. no need to add stuff to jar really